### PR TITLE
Modifying the bootstrap init so that a type can be passed in.

### PR DESCRIPTION
**Pull request by hub-cap**
- Moved bootstrap_init to bootstrap_init-mysql
- Modified redstack to work with multiple image types

### CODE REVIEW

**Comment by cp16net** [comment]
+1

File: scripts/redstack [hunk 1]
```diff
@@ -579,15 +583,22 @@ function build_vm() {
...
     # man ubuntu-vm-builder for more infomration on the --copy flag
     echo "$HOMEDIR/.ssh/id_rsa.pub $HOMEDIR/.ssh/id_rsa.pub" >> $COPY_FILE
     echo "$HOMEDIR/.ssh/id_rsa $HOMEDIR/.ssh/id_rsa" >> $COPY_FILE
     echo "$HOMEDIR/.ssh/authorized_keys $HOMEDIR/.ssh/authorized_keys" >> $COPY_FILE
 
+    if [ $SERVICE_TYPE == 'mysql' ]
+    then
+        PKGS="--addpkg mysql-server-5.1"
+    else
+        exclaim "Warning: You did not pass in a valid image type, the image may fail"
```
Comment (TimSimpsonR) [inline scripts/redstack:596]: I say kill it here with "exit". We'll add other cases later.

Comment (TimSimpsonR) [inline scripts/redstack:596]: N/m, I was blind and didn't see it.

**Comment by TimSimpsonR** [comment]
+1

**Comment by ed-** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Pull request merged**